components/resources/wallet_strings.grdp

```diff
@@ -1041,4 +1041,7 @@
   <message name="IDS_BRAVE_WALLET_UNTRUSTED_REQUEST_WARNING" desc="Warning that the pending request may be malicious">This request appears malicious, and may leave you vulnerable to theft. Do not continue unless you trust the app.</message>
   <message name="IDS_BRAVE_WALLET_PROCEED_BUTTON" desc="Proceed action button text">Proceed</message>
   <message name="IDS_BRAVE_WALLET_FOUND_ISSUES" desc="Notice that a number of issues were found">We found <ph name="VALUE">$1<ex>2</ex></ph> issues.</message>
+  <message name="IDS_BRAVE_WALLET_ACCOUNT_NAME_TOO_LONG_ERROR" desc="An error that appears when the user enters an account name that is too long">Account name must be 30 characters or less</message>
```

Comment:
Should this use the constant value so if we later change from the 30 character limit we don't need to update the string too? I was just making iOS changes to do this (as well as validating with the new mojo constant) on the [localized string for iOS](https://github.com/brave/brave-core/blob/ae0a5007d37f591bf6b8c664d1a80c7699de2947/ios/brave-ios/Sources/BraveWallet/WalletStrings.swift#L190-L197).

Sorry @mkarolin, bad timing right after your review 😅